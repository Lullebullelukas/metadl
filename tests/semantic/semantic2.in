A("", ""), B("") :- NEQ(0, 0).
A(x, y) :- B(x).
