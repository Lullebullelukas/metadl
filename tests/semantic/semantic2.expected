Unbound variable(s): [y] in A(x, y) :- 
	B(x).

